push_notifications/api/rest_framework.py

```diff
@@ -28,19 +28,6 @@ def to_representation(self, value):
 		return value
 
 
-# Serializers
```

Comment:
why you deleted this part?

Reply:
I thought this part didn't need for your prodject.
I am  a senior Ptthon/ Django developer.